datasets/ge/ot_list/crawler.py

```diff
@@ -46,7 +46,7 @@ def crawl_row(context: Context, row: Dict[str, str]):
     context.log.debug(f"Unique ID {person.id}")
     if birth_date != UNKNOWN:
         person.add("birthDate", h.parse_date(birth_date, DATE_FORMATS))
-    person.add("topics", "sanction")
+    person.add("topics", ["sanction", "crime.war"])
```

Comment:
We'd like to use `crime.war` very judiciously so lets leave this one to just `sanction`. Totally see the reasoning though.

Reply:
Makes sense, I'll revert it!